src/main/resources/templates/fragments/login_navigation.html

```diff
@@ -3,24 +3,6 @@
     <head>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a 'favicon' declaration in this 'header' tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)